src/main/scala/com/monsanto/arch/cloudformation/model/resource/ECS.scala

```diff
@@ -137,14 +142,22 @@ object ContainerDefinition extends DefaultJsonProtocol {
                                                 LogConfiguration: Option[LogConfiguration] = None,
                                                 Memory: M = None,
                                                 MemoryReservation: R = None,
+                                                MountPoints: Option[Seq[MountPoint]] = None,
                                                 Name: String,
                                                 PortMappings: Option[Seq[PortMapping]] = None,
                                                 Privileged: Option[Boolean] = None,
                                                 ReadonlyRootFilesystem: Option[Boolean] = None,
                                                 Ulimits: Option[Seq[Ulimit]] = None,
                                                 User: Option[Token[String]] = None,
                                                 VolumesFrom: Option[Seq[VolumesFrom]] = None,
-                                                WorkingDirectory: Option[Token[String]] = None)(implicit ev1: MemoryRequirement[M, R]): ContainerDefinition = ContainerDefinition(Command, Cpu, DisableNetworking, DnsSearchDomains, DnsServers, DockerLabels, DockerSecurityOptions, EntryPoint, Environment, Essential, ExtraHosts, Hostname, Image, Links, LogConfiguration, Memory, MemoryReservation, Name, PortMappings, Privileged, ReadonlyRootFilesystem, Ulimits, User, VolumesFrom, WorkingDirectory, None)
+                                                WorkingDirectory: Option[Token[String]] = None)(implicit ev1: MemoryRequirement[M, R]): ContainerDefinition =
+    ContainerDefinition(
+      Command, Cpu, DisableNetworking, DnsSearchDomains, DnsServers, DockerLabels, DockerSecurityOptions,
+      EntryPoint, Environment, Essential, ExtraHosts, Hostname, Image, Links, LogConfiguration, Memory,
+      MemoryReservation, MountPoints, Name, PortMappings, Privileged, ReadonlyRootFilesystem, Ulimits, User,
+      VolumesFrom, WorkingDirectory, None
+    )
+
   implicit val format: JsonFormat[ContainerDefinition] = new RootJsonFormat[ContainerDefinition] {
     override def write(cd: ContainerDefinition) = {
       val obj = JsObject(
```

Comment:
@jfklingler I haven't had a chance to confirm myself, can you respond to @Tvaroh 's comment?